app.py

```diff
@@ -79,31 +132,41 @@ def post_to_discord(channel, text):
              json={"content": text},
              headers={"Authorization": "Bot " + config["token"],
                       "User-Agent": "gitlab-discord-bridge by blha303"
-                     }).json()
+                      }).json()
     return "cool"
 
-@app.route('/<channelid>', methods=['GET', 'POST'])
+
+@app.route('/<channelid>', methods=['POST'])
 def index(channelid):
     if request.method != "POST":
-        return make_response("lol", 400)
-    if ("host" in config and request.remote_addr != config["host"]) or request.headers.get("X-Gitlab-Token", "") != config["secret"]:
-        return make_response("go away please", 403)
+        return make_response("Method Not Allowed", 405)
+    if ("host" in config and request.remote_addr != config["host"]) or request.headers.get("X-Gitlab-Token", "") != \
+            config["secret"]:
```

Comment:
Unnecessary line split, please remove

Reply:
Well https://www.python.org/dev/peps/pep-0008/ does recommend even shorter line length than PyCharm formating does, but yeah this one is truly unnecessary.